packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md

```diff
@@ -7,29 +7,836 @@ category: management
 
 <ApiLifecycle access="ea" />
 
+> **Note:** The MyAccount API is being enhanced, accessible at `/api/idp/myaccount`. Use this endpoint if you plan to use the MyAccount API. The existing endpoint (`/api/v1/myaccount`) is deprecated.
+>
+>Also, to use the new endpoint, move the `version` from the path to the `Accept` header. Use a value of `application/json; okta-version=1.0.0`.
+
 The Okta MyAccount API allows end users (with or without administrator access) to fetch and update their own Okta user profiles.  It implements a subset of the existing [Users API](/docs/reference/api/users/) but with significant differences.  This API does not expose information an end user should not have access to, and it does not support lifecycle operations.
 
-All operations in this API implicitly refer to the user making the API call.  No user ID is needed (or even accepted).
+All operations in this API implicitly refer to the user making the API call.  No user ID is needed (or even accepted).
+
+<!--
+## Get started
+
+Explore the MyAccount API:
+
+-->
+
+## MyAccount Operations
+
+The MyAccount API has the following operations:
+
+* [Get My Emails](#get-my-emails)
+* [Get My Email](#get-my-email)
+* [Add My Email](#add-my-email)
+* [Challenge My Email](#challenge-my-email)
+* [Verify My Email](#verify-my-email)
+* [Poll My Email Challenge](#poll-my-email-challenge)
+* [Delete My Email](#delete-my-email)
+* [Add My Phone](#add-my-phone)
+* [Get My Phone](#get-my-phone)
+* [Get My Phones](#get-my-phones)
+* [Delete My Phone](#delete-my-phone)
+* [Send Phone challenge](#send-phone-challenge)
+* [Verify Phone challenge](#verify-phone-challenge)
```

Comment:
Per Zeus' comment in Jay's original PR we might want to reorder the operations for phone so that it is consistent with email:
```
Get My Phones
Get My Phone
Add My Phone
Challenge My Phone
Verify My Phone
Delete My Phone
```